identity/README.md

```diff
@@ -261,6 +261,44 @@ message Capability {
     Type type = 1;
   }
 
+  // VolumeGroup contains the features of the volumegroup operation
+  // that the CSI-driver supports.
+  message VolumeGroup {
+    // Type describes a CSI Service that CSI-drivers can support.
+    enum Type {
+      // UNKNOWN indicates that the CSI-driver does not support the
+      // VolumeGroup operation in the current mode. The CSI-Addons CO
+      // plugin will most likely ignore this node for the
+      // VolumeGroup operation.
+      UNKNOWN = 0;
+
+      // VOLUME_GROUP indicates that the CSI-driver provides RPCs for a
+      // VolumeGroup operation.
+      // The presence of this capability determines whether the CSI-Addons CO
+      // plugin can invoke RPCs that require access to the storage system,
+      // similar to the CSI Controller (provisioner).
+      VOLUME_GROUP = 1;
+      // LIMIT_VOLUME_TO_ONE_VG indicates that the CSI-driver does not support
+      // that one volume will be in multiple VGs, so it can be in only one VG.
+      LIMIT_VOLUME_TO_ONE_VG = 2;
+      // DO_NOT_ALLOW_VG_TO_DELETE_VOLUMES indicates that the CSI-driver
+      // does not support that deletion of the VG 
+      // won't delete the volumes under it.
+      DO_NOT_ALLOW_VG_TO_DELETE_VOLUMES = 3;
+      // MODIFY_GROUP_MEMBERSHIP indicates that the CSI-driver
+      // supports that nodifying the VG.
+      MODIFY_GROUP_MEMBERSHIP = 4;
```

Comment:
because the RPC call that it will enable, is `ModifyVolumeGroupMembershipResponse`, what do you think better? `MODIFY_VOLUME_GROUP` or `MODIFY_VOLUME_GROUP_Membership`?

Reply:
MODIFY_VOLUME_GROUP looks good to me.

Reply:
done